apps/web/src/utils/transactions.ts

```diff
@@ -19,12 +19,12 @@ import {
   isTransferTxInfo,
   isTxQueued,
 } from './transaction-guards'
-import type { MetaTransactionData } from '@safe-global/safe-core-sdk-types/dist/src/types'
-import { OperationType } from '@safe-global/safe-core-sdk-types/dist/src/types'
+import type { MetaTransactionData } from '@safe-global/types-kit/dist/src/types'
+import { OperationType } from '@safe-global/types-kit/dist/src/types'
 import { getReadOnlyGnosisSafeContract } from '@/services/contracts/safeContracts'
 import extractTxInfo from '@/services/tx/extractTxInfo'
 import type { AdvancedParameters } from '@/components/tx/AdvancedParams'
-import type { SafeTransaction, TransactionOptions } from '@safe-global/safe-core-sdk-types'
+import type { SafeTransaction, TransactionOptions } from '@safe-global/types-kit'
```

Comment:
Imports referencing `dist` folder are not needed. Those types can be fetch from the package root like in 

https://github.com/safe-global/safe-wallet-monorepo/pull/5910/files#diff-8626d5cfc89b907a904ed95f758dfbb947296904c06240e51d20d4b060cf6029R6

It's should also be possible to import the types with just one import sentence